Makefile

```diff
@@ -236,7 +236,16 @@ YQ_VERSION ?= v4.45.3
 .PHONY: kustomize
 kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
 $(KUSTOMIZE): $(LOCALBIN)
-	$(call go-install-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v5,$(KUSTOMIZE_VERSION))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(KUSTOMIZE)) ;\
+	OS=$$(go env GOOS) && ARCH=$$(go env GOARCH) && \
+	TMPFILE=$$(mktemp) && \
+	curl -sSLo $$TMPFILE https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/$(KUSTOMIZE_VERSION)/kustomize_$(KUSTOMIZE_VERSION)_$${OS}_$${ARCH}.tar.gz && \
+	tar -xzf $$TMPFILE -C $(dir $(KUSTOMIZE)) && \
+	rm $$TMPFILE && \
```

Comment:
Nit: this could be a single command if you pipe the curl to tar. And the TMPFILE won't be needed either.

Reply:
Can we use the same technique like `go-install-tool` that creates a `bin/kustomize-<version>` binary and then a symbolic link to `bin/kustomize`. That way it is ensured that you always have the kustomize version running that is mentioned in the Makefile.

Reply:
I mean something like

```
.PHONY: kustomize
kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
$(KUSTOMIZE): $(LOCALBIN)
	@[ -f "$(KUSTOMIZE)-$(KUSTOMIZE_VERSION)" ] || { \
	set -e; \
	echo "Downloading kustomize $(KUSTOMIZE_VERSION)" ;\
	rm -f $(KUSTOMIZE) || true ;\
	OS=$$(go env GOOS) && ARCH=$$(go env GOARCH) ;\
	curl -sSLf https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/$(KUSTOMIZE_VERSION)/kustomize_$(KUSTOMIZE_VERSION)_$${OS}_$${ARCH}.tar.gz | tar xz -C $(dir $(KUSTOMIZE)) ;\
	mv $(dir $(KUSTOMIZE))kustomize $(KUSTOMIZE)-$(KUSTOMIZE_VERSION) ;\
	chmod +x $(KUSTOMIZE)-$(KUSTOMIZE_VERSION) ;\
	} ;\
	ln -sf $(KUSTOMIZE)-$(KUSTOMIZE_VERSION) $(KUSTOMIZE)
```